### ISSUE TITLE
Unable to add mod

### ISSUE THREAD

**Issue by AsukaSoryu0** [Jorixon/JASM#308]
View the related log file, is probably insufficient permission? This has never happened before
![1](https://github.com/user-attachments/assets/19135bbb-7bc7-4cd3-9dc9-f9a795a4eb45)
![2](https://github.com/user-attachments/assets/e02dc360-e986-45f7-a848-794d3b4c797e)

**Comment by Jorixon** [comment]
Hi 
I belive the issue happens because the mods are stored in the Program Files folder. I think apps (or jasm in this case) needs admin permissions to access files in the program files folder.

Could try running JASM as administrator to see if it fixes it, but it's not something I'd recommend as a permanent fix.

**Comment by AsukaSoryu0** [comment]
> Hi I belive the issue happens because the mods are stored in the Program Files folder. I think apps (or jasm in this case) needs admin permissions to access files in the program files folder.
> 
> Could try running JASM as administrator to see if it fixes it, but it's not something I'd recommend as a permanent fix.

Oddly enough, it used to be possible to add mods. Is it a win11 24h2 problem?

**Comment by farsdewibs0n** [comment]
> Oddly enough, it used to be possible to add mods. Is it a win11 24h2 problem?
Yes, most likely, it's a permission issue. The mod folder possibly isn't made by the same user as the one you're currently using. Could be happened because you reinstalled your windows (or get the mods from someone else's comptuer).

Try these solution: 
https://www.tenforums.com/tutorials/3841-add-take-ownership-context-menu-windows-10-a.html 
https://www.askvg.com/add-take-ownership-option-in-file-folder-context-menu-in-windows-vista/
(both are basically the same solution. Take ownership of ZZMI folder.)
